control/content/index.html

```diff
@@ -3,263 +3,124 @@
 <head lang="en">
     <meta charset="UTF-8">
 
-	<!-- CSS -->
+    <!-- CSS -->
     <link href="../../../../styles/helper.css" rel="stylesheet">
+    <link href="../../resources/peopleplugin.css" rel="stylesheet">
 
-	<!-- JS -->
+    <!-- JS -->
+    <script src="../../../../scripts/jquery/jquery-1.11.2.min.js"></script>
+    <script src="../../resources/bootstrap.min.js"></script>
```

Comment:
For data-toggle on bootstrap dropdown we need bootstrap-ui.js
Refer to http://getbootstrap.com/javascript/#dropdowns